documentation review - part 1 (#1062)
FoundationDB/fdb-kubernetes-operator: README.md

You can test your setup with `SKIP_TEST=1 make container-build` which will build the image locally.
After the command successfully finished you can verify with `nerdctl -n k8s.io images fdb-kubernetes-operator:latest` that the image is available.

> ### Known Challenges

Reviewer: ```suggestion
### Known Limitations

```

> ### Known Limitations
> 1. Support for backups in the operator is still in development, and there are significant missing features.

Reviewer: ```suggestion
### Known Limitations

1. Support for backups in the operator is still in development, and there are significant missing features.
```

2. The unified image is still experimental, and is not recommended outside of development environments.

> 3. Read more challenges in [Warnings](docs/manual/warnings.md)

Reviewer: ```suggestion
3. Additional limitations can be found under [warnings](docs/manual/warnings.md).
```
